datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/eula.py

```diff
@@ -46,7 +52,8 @@ def eula(check):
             if b'%PDF' not in magic_number:
                 echo_info(f'{check_name}... ', nl=False)
                 echo_info(' FAILED')
-                echo_failure(f'  {eula_relative_location} is not a PDF file')
+                message = f'{eula_relative_location} is not a PDF file'
+                echo_failure('  ' + message)
                 failed_checks += 1
```

Comment:
Should this also annotate?
```suggestion
                annotate_error(manifest_file, message)
                failed_checks += 1
```